client/web/src/components/exit-node-selector.tsx

```diff
@@ -74,72 +76,88 @@ export default function ExitNodeSelector({
           {
-            "border-gray-200": none,
-            "bg-amber-600 border-amber-600": advertising,
-            "bg-blue-500 border-blue-500": using,
+            "bg-red-600": offline,
           },
           className
         )}
       >
-        <button
-          className={cx("flex-1 px-2 py-1.5 rounded-[1px]", {
-            "bg-white hover:bg-stone-100": none,
-            "bg-amber-600 hover:bg-orange-400": advertising,
-            "bg-blue-500 hover:bg-blue-400": using,
-            "cursor-not-allowed": disabled,
+        <div
+          className={cx("p-1.5 rounded-md border flex items-stretch gap-1.5", {
+            "border-gray-200": none,
+            "bg-yellow-300 border-yellow-300": advertising,
+            "bg-blue-500 border-blue-500": using,
+            "bg-red-500 border-red-500": offline,
           })}
-          onClick={() => setOpen(!open)}
-          disabled={disabled}
         >
-          <p
-            className={cx(
-              "text-gray-500 text-xs text-left font-medium uppercase tracking-wide mb-1",
-              { "bg-opacity-70 text-white": advertising || using }
-            )}
-          >
-            Exit node
-          </p>
-          <div className="flex items-center">
-            <p
-              className={cx("text-gray-800", {
-                "text-white": advertising || using,
-              })}
-            >
-              {selected.Location && (
-                <>
-                  <CountryFlag code={selected.Location.CountryCode} />{" "}
-                </>
-              )}
-              {selected === runAsExitNode
-                ? "Running as exit node"
-                : selected.Name}
-            </p>
-            <ChevronDown
-              className={cx("ml-1", {
-                "stroke-neutral-800": none,
-                "stroke-white": advertising || using,
-              })}
-            />
-          </div>
-        </button>
-        {(advertising || using) && (
           <button
-            className={cx("px-3 py-2 rounded-sm text-white", {
-              "bg-orange-400": advertising,
-              "bg-blue-400": using,
+            className={cx("flex-1 px-2 py-1.5 rounded-[1px]", {
+              "bg-white hover:bg-stone-100": none,
+              "bg-yellow-300 hover:bg-yellow-200": advertising,
+              "bg-blue-500 hover:bg-blue-400": using,
+              "bg-red-500 hover:bg-red-400": offline,
               "cursor-not-allowed": disabled,
             })}
-            onClick={(e) => {
-              e.preventDefault()
-              e.stopPropagation()
-              handleSelect(noExitNode)
-            }}
+            onClick={() => setOpen(!open)}
             disabled={disabled}
           >
-            Disable
+            <p
+              className={cx(
+                "text-gray-500 text-xs text-left font-medium uppercase tracking-wide mb-1",
+                { "bg-opacity-70 text-white": advertising || using || offline }
```

Comment:
The `|| offline` here isn't totally necessary I think? Since one of the first two will always be true

Reply:
done